pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -170,6 +178,36 @@ func getClients() (*vCenterClient, error) {
 	}, nil
 }
 
+// checkComputeResource checks the ClusterComputeResource to see if there are any attached hosts.  Without
+// hosts, the check for networks will fail.  We want the message to be clear if this is the cause so
+// users know what to address.  This only returns an error if unable to verify or if no ComputeResources
+// are found.
+func checkComputeResource(ctx context.Context, datacenter string, cluster string, finder Finder) error {
+	ctx, cancel := context.WithTimeout(ctx, 60*time.Second)
+	defer cancel()
+
+	path := fmt.Sprintf("/%s/host/%s", datacenter, cluster)
+	ccr, err := finder.ClusterComputeResource(context.TODO(), path)
+	if err != nil {
+		return err
+	}
+
+	var ccrMo mo.ComputeResource
+	logrus.Debugf("CCR: %v", ccr)
+
+	err = ccr.Properties(ctx, ccr.Reference(), []string{"host"}, &ccrMo)
+	logrus.Debugf("Hosts: %v", ccrMo.Host)
+	if err != nil {
+		return err
+	}
+	if len(ccrMo.Host) == 0 {
+		logrus.Errorf("No hosts found in cluster %v. Please make sure Compute Resources are assigned to the cluster before installation.", cluster)
+		return fmt.Errorf("no hosts found in cluster %v", cluster)
```

Comment:
Why do we need both a message logged and an error here saying basically the same thing?

Reply:
My original thought was to make sure we had enough info to instruct user when to have this fix.  I didn't want to return the longer string because it makes the overall error really long and unsightly.  I can remove this message.